méli-mélo/th-canadaday/style.css

```
@@ -0,0 +1,35 @@
+/* Thematic red */
+.bg-canadaday,
+body .bg-canadaday {
+    background-color: #990000;
+}
+
+.panel-canadaday,
+hr.brdr-canadaday {
+    border-color:#990000;
+}
+
+/* Hover and focus state */
+a.bg-canadaday:hover,
+a.bg-canadaday:focus,
+button.bg-canadaday:hover,
+button.bg-canadaday:focus {
+    background-color: #6b0303;
+}
+
+/* Active state */
+a.bg-canadaday:active,
+a.bg-canadaday:active:focus,
+a.bg-canadaday.active:focus,
+a.bg-canadaday.active.focus,
+a.bg-canadaday:active.focus,
```

Do you have a demo where it use the CSS class `active` and `focus` on anchor along with the `bg-canadaday` CSS class?

As you mentioned, you wanted to avoid the primary blue, my local test showed me it is when the anchor state is at the same time `:active`  and `:hover`. 
I propose you the following update instead

```suggestion
a.bg-canadaday:active:hover,
```